build-system/tasks/e2e/e2e-types.js

```diff
@@ -52,6 +52,7 @@ const Key = {
   'ArrowLeft': 'ArrowLeft',
   'ArrowRight': 'ArrowRight',
   'ArrowUp': 'ArrowUp',
+  'Backspace': 'Backspace',
```

Comment:
Necessary to trigger a backspace. I don't like this double mapping :(

Reply:
Agreed :(